client/my-sites/patterns/index.node.tsx

```diff
@@ -20,6 +25,9 @@ function renderPatterns( context: RouterContext, next: RouterNext ) {
 			categoryGallery={ CategoryGalleryServer }
 			isGridView={ !! context.query.grid }
 			patternGallery={ PatternGalleryServer }
+			patternTypeFilter={
```

Comment:
I had doubts regarding props drilling for `isGridView`, and now I think more and more - maybe let's consider some option to create some helper function for retrieving `params.type` from URL?
Also would be cool to take `grid`-query directly. We will have more simple component, w/o extra props.
WDYT?